src/lib/logger.ts

```diff
@@ -1,7 +1,13 @@
-import { logger } from '@libp2p/logger'
+import { prefixLogger, type ComponentLogger } from '@libp2p/logger'
 
-const logObj = logger('helia:service-worker-gateway')
+const host = globalThis.location.host.replace(':', '_')
 
-export const log = logObj
-export const error = logObj.error
-export const trace = logObj.trace
+const swLogPrefix = `helia:service-worker-gateway:sw:${host}`
```

Comment:
```suggestion
const swLogPrefix = `helia:sw-gateway:sw:${host}`
```

Any thoughts on making the logs a little shorter since we use sw as an acception abbreviation elsewhere too?

Reply:
Yeaaa i thought about that. Theyre pretty long, so if you think shorter is fine im absolutely a fan.

I think uniqueness is the most important here, instead of verbosity. If someone is looking at these logs, theyre likely aware its a "service-worker-gateway"

Reply:
Is "helia:gateway:sw" & "helia:gateway:ui" too obtuse?